charters/DataProcessingService/DataProcessingService.md

```diff
@@ -0,0 +1,45 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
```

Comment:
Per the process, this file is supposed to be named charter.md.

Reply:
thanks!